### ISSUE TITLE
Allow servlet headers to be disabled individually

### ISSUE THREAD

**Issue by eleftherias** [spring-projects-experimental/spring-security-kotlin-dsl#9]
Currently the way to disable cache control headers, in a servlet application, is to disable the defaults and then add the desired headers one by one.
The situation is the same for frame options, XSS and other headers.
A user should be able to disable a single group of headers without having knowledge of the other header options.

For example, disabling cache control headers would look like this:
 ```
 @EnableWebSecurity
 class SecurityConfig : WebSecurityConfigurerAdapter() {
  override fun configure(http: HttpSecurity) {
      http {
          headers {
              cacheControl {
                  disable()
              }
          }
      }
  }
 }
 ```

**Comment by d-stoll** [comment]
Hey @eleftherias, i really like the idea of the project. Can i work on this?

**Comment by eleftherias** [comment]
Thanks @d-stoll, the issue is yours! Feel free to reach out if you have any questions.

**Comment by eleftherias** [comment]
Fixed via 3b4f759386714f4a35593ec0705e8bca4aad5e7c.
